command/root/root.go

```diff
@@ -0,0 +1,63 @@
+package root
```

Comment:
Why not put this in the `command` package itself? Having a `root` package kinda breaks the pretty package structure around subcommands

Reply:
Moving this file in the `command` package would create a cyclic import, that's why it's in a separate package

Reply:
Understood, thanks!